
syntax = "proto3";
package common;

message ModelBlockId {
    string model_name = 1;
    string server_id = 2;
    string block_idx = 3;
}

message Empty {}

message OptimizedPlan {
    map<string, string> plans_map = 1;
}